### ISSUE TITLE
Filesystem scans not working (Tested on Mac M1 & Ubuntu EC2)

### ISSUE THREAD

**Issue by rohitcoder** [trufflesecurity/trufflehog#739]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### TruffleHog Version
3.9.0

### Trace / Debug Output
```
DEBU[0000] running version 3.9.0                        
2022/08/28 10:53:56 [updater parent] run
2022/08/28 10:53:56 [updater parent] checking for updates...
2022/08/28 10:53:57 [updater parent] failed to get latest version: already up to date
2022/08/28 10:53:57 [updater parent] starting /opt/homebrew/bin/trufflehog
DEBU[0000] running version 3.9.0                        
2022/08/28 10:53:57 [updater child#1] run
2022/08/28 10:53:57 [updater child#1] start program
trufflehog 3.9.0
DEBU[0000] running with up to 8 workers                 
DEBU[0000] loaded 2 decoders                            
DEBU[0000] loaded 694 detectors total, 694 with verification enabled. 0 with verification disabled 
INFO[0000] starting pprof and fgprof server on :18066 /debug/pprof and /debug/fgprof 
🐷🔑🐷  TruffleHog. Unearth your secrets. 🐷🔑🐷

2022/08/28 10:53:58 [updater parent] checking for updates...
DEBU[0001] scanned 619 chunks                           
2022/08/28 10:53:58 [updater parent] prog exited with 0
```

### Expected Behavior
After running ``trufflehog filesystem --directory=/Users/<user>/Desktop/repo --debug`` this should return some secretes like we get through git repo links

### Actual Behavior
Nothing is getting returned, even if i add AWS Keys or other keys in project.

### Steps to Reproduce

 1. Clone any repo with sensitive in that
 2. Run trufflehog filesystem --directory=<PROJECT_PATH> --debug

## Environment
 * OS: MacOs Monterey
 * Version 12.4

**Comment by bill-rich** [comment]
I'm seeing the same thing and looking into it.

**Comment by dustin-decker** [comment]
This should be fixed in v3.10!

https://github.com/trufflesecurity/trufflehog/releases/tag/v3.10.0

**Comment by aviadhahami** [comment]
Using the `latest`, re-seeing this. Is anyone seeing it? (M1)
